docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+- [Platform improvements](#platform-improvements)
+- [API changes](#api-changes)
+- [Visual Studio integration](#visual-studio-integration)
+- [Documentation improvements](#documentation-improvements) 
+
+## Tooling
+
+**`dotnet restore` runs implicitly**
+
+In previous versions of the .NET Core, you had to run the [dotnet restore](../tools/dotnet-restore.md) command to download dependencies immediately after you created a new project with the [dotnet new](../tools/dotnet-new) command, as well as whenever you added a new dependency to your project. In .NET Core 2.0, `dotnet restore` runs implicitly when the `dotnet new` command executes, as well as if dependencies need to be updated when the `run`, `build`, and `publish` commands execute.
+
+## Language support
+
+In addition to supporting C# and F#, .NET Core 2.0 also supports Visual Basic.
+
+**Visual Basic**
+
+With version 2.0, .NET Core now supports Visual Basic 2017. For example, to create a simple Visual Basic "hello world" application, do the following from the command line:
+
+1. Open a console window, create a directory for your project and make it the current directory.
+
+1. Enter the command `dotnet new console -lang vb`.
+
+   This creates a project file with a `.csproj` file extension, along with a Visual Basic source code file named `program.vb`. This file contains the source code to write the string "Hello World!" to the console window.
+  
+1.  Enter the command `dotnet run`. The [.NET Core CLI tools](../tools/) automatically compile and execute the application, which displays the message "Hello World!" in the console window.
+
+Visual Studio Preview 15.3 also includes templates for Visual Basic that let you create .NET Core console apps, class libraries, unit test projects, and xUnit test projects.
+
+**Support for C# 7.1**
+
+.NET Core 2.0 supports C# 7.1, which adds a number of new features, including:
+
+- The `Main` method, the application entry point, can be marked with the [async](../../csharp/language-reference/keywords/async.md) keyword.
+- Inferred tuple names.
+- Default expressions.
+- Pattern matching with generics. 
+
+<!-- For more information see [link to C# what's new](url). -->
+
+## Platform improvements
+
+.NET Core 2.0 includes a number of features that make it easier and less cumbersome to install .NET Core and to use it on supported operating systems.
+
+**.NET Core for Linux is a single implementation**
+
+.NET Core 2.0 offers a single Linux implementation that works on multiple Linux distributions. .NET Core 1.x required that yu download a distribution-specific Linux implementation.
+
+You can also develop apps that target Linux as a single operating system. .NET Core 1.x required that you target each Linux distribution separately. 
+
+**Support for the Apple cryptographic libraries**
+
+.NET Core 1.x on macOS required the OpenSSL toolkit's cryptographic library. .NET Core 2.0 uses the Apple cryptographic libraries and does not require OpenSSL. 
```

Comment:
I'd make it clearer that you don't need to install OpenSSL anymore on macOS.  and remember the contractions: does not ->doesn't